### PR TITLE
blog: add release notes for 3.6.1

### PR DESCRIPTION
**Pull request by kayx23**
Add release notes for 3.6.1. Not yet formally released (updated expected date: Nov 1).

### CODE REVIEW

File: blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: The Apache APISIX 3.6.1 version is released on October 17, 2023. This release includes a few bug fixes.
+tags: [Community]
+---
+
+We are glad to present Apache APISIX 3.6.1 with a number of bug fixes and continuous improvements to user experiences.
```
Comment (yzeng25) [inline blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md]:
```suggestion
> We are glad to present Apache APISIX 3.6.1 with a number of bug fixes and continuous improvements to user experiences.
```

File: blog/zh/blog/2023/10/17/release-apache-apisix-3.6.1.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: Apache APISIX 3.6.1 版本于 2023 年 10 月 17 日发布。该版本带来了一系列修复。
+tags: [Community]
+---
+
+我们很高兴地宣布 Apache APISIX 3.6.1 版本已经发布，带来一系列修复，持续优化用户体验。
```
Comment (yzeng25) [inline blog/zh/blog/2023/10/17/release-apache-apisix-3.6.1.md]:
```suggestion
> 我们很高兴地宣布 Apache APISIX 3.6.1 版本已经发布，带来一系列修复，持续优化用户体验。
```

File: blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md (hunk 1 above)
Comment (kayx23) [reply blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md]: Thanks for the review. But why?

Comment (yzeng25) [reply blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md]: - To tell apart the summary and content on the blog page, i.e. 

image

- To remind writers don't forget to add `<!--truncate-->` after summary.

**Review by yzeng25 — CHANGES_REQUESTED** [review]
`blog/zh/blog/2023/11/release-apache-apisix-3.6.1.md` -> `blog/zh/blog/2023/11/01/release-apache-apisix-3.6.1.md`

**Review by Revolyssup — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Comment by yzeng25** [comment]
@SkyeYoung @guoqqqi  this pull request caused github actions failed, can you check on it?

**Comment by yzeng25** [comment]
> @SkyeYoung @guoqqqi this pull request caused github actions failed, can you check on it?

fixed in https://github.com/apache/apisix-website/pull/1720

**Pull request merged**